zk/src/archive/web/js/zk/au.js

```diff
@@ -1000,7 +1000,7 @@ zAu.beforeSend = function (uri, req, dt) {
 				}
 			}
 			if (!reqInf.ignorable && !zk.unloading) {
-				var msg = req.statusText;
+				var msg = reqInf.statusText;
```

Comment:
this is an even more important fix, we should also have a test case for the default client side error handling, besides the custom error-uri config